apps/marketplace/pages/BriefResponsePage.js

```diff
@@ -60,8 +60,13 @@ class BriefResponsePage extends Component {
     const briefResponseType = this.props.match.params.briefResponseType
     if (briefId && briefResponseId) {
       this.resetForm()
-      this.props.loadInitialData(briefId).then(
-        this.props.loadBriefResponse(briefResponseId).then(response => {
+      this.props.loadInitialData(briefId).then(briefResponse => {
+        if (briefResponse.data.brief.status !== 'live') {
+          this.props.setError('This opportunity has closed - you can no longer make edits.')
```

Comment:
What about withdrawn opportunities? Does it make sense to rewrite this to something like the following?

`Edits can't be made on a <status> opportunity`